src/main/java/upeldev/com/github/upel3/controllers/new_controllers/NewActivityController.java

```diff
@@ -48,13 +48,17 @@ public String createCourse(
             @PathVariable("id") Long id,
             @RequestParam(value = "activityName") String activityName,
             @RequestParam(value = "activityDescription") String activityDescription,
-            @RequestParam(value = "passValue")  int passValue,
+            @RequestParam(value = "passValue", required=false) Integer passValue,
+            @RequestParam(value = "pass")  String passOption,
             @RequestParam(value = "subActivity")  String[] subActivityName,
             @RequestParam(value = "maxPoints")  int[] maxPoints,
             Model model,
             Principal principal) {
 
-
+        System.out.println(passValue);
```

Comment:
This sout should be removed.

Reply:
I think that the progress bar should stay green if the student got enough points, even if he didn't pass all activities because it indicates the points obtained in whole course, not in individual activities. If student failed activities - and failed course he will see it.

Reply:
I've deleted System.out.println(passValue) line